### PR TITLE
BatchRenormalization / ROR Removal

### PR DESCRIPTION
**Pull request by titu1994**
Remove BatchRenormalization, Residual-of-Residual Networks application and related tests

### CODE REVIEW

**Comment by titu1994** [comment]
@gabrieldemarmiesse As per comments in https://github.com/keras-team/keras-contrib/issues/415

File: contrib_docs/structure.py [hunk 1]
```diff
@@ -1,94 +1,93 @@
...
+    'serialize',
+    'deserialize',
+    'get',
+    'set_image_dim_ordering',
+    'normalize_data_format',
+    'image_dim_ordering',
+    'get_variable_shape',
+    'Constraint'
+}
+
```
[the remark this answers is not in the document]
Comment (titu1994) [reply contrib_docs/structure.py]: Hmm PyCharm cant seem to fix it. I set it to use Unix line endings and redid the commit, but it seems it isnt being corrected.

[the remark this answers is not in the document]
Comment (gabrieldemarmiesse) [reply contrib_docs/structure.py]: weird, my pycharm doesn't do this. Are you on windows?

[the remark this answers is not in the document]
Comment (titu1994) [reply contrib_docs/structure.py]: Yes im on Windows. That might be the cause.

Edit: Tried to fix it with another commit using Unix line endings inside PyCharm, but it doesnt seem to make a difference for some reason. The other commits show no such issues.

[the remark this answers is not in the document]
Comment (gabrieldemarmiesse) [reply contrib_docs/structure.py]: I'm on windows with WSL and my pycharm doesn't do that. Weird. If you have notepad++ it does a really good job at changing line endings, much better than pycharm for this. Can you try?

[the remark this answers is not in the document]
Comment (titu1994) [reply contrib_docs/structure.py]: No change. How odd. Never experienced such an issue prior to this any commit. Notepad ++ changed to Unix Line encodings as well, but git registers this as a full change for some reason.

**Comment by titu1994** [comment]
Closed via https://github.com/keras-team/keras-contrib/pull/424

**Pull request closed without merging**